lib/rules/sotd/stability.js

```diff
@@ -4,21 +4,23 @@
 // draft document and may be updated, replaced or obsoleted by other documents at any time. It is
 // inappropriate to cite this document as other than work in progress.</p>
 const util = require('../../util');
-function findSW (candidates, sr) {
-    var sw = null;
+function findSW(candidates, sr) {
+    var crType = sr.config.crType
+    ,   INTRO_S = "A Candidate Recommendation Snapshot has received wide review and is intended to gather implementation experience."
+    ,   INTRO_D = "A Candidate Recommendation Draft integrates changes from the previous Candidate Recommendation that the Working Group intends to include in a subsequent Candidate Recommendation Snapshot."
```

Comment:
```suggestion
    ,   INTRO_D = "A Candidate Recommendation Draft is published changes from the previous Candidate Recommendation that the Working Group intends to include in to solicit review of intended changes from the previous Candidate Recommendation Snapshot."
```

Change in accordance with  https://github.com/w3c/tr-design/pull/209/files